ui/src/services/salt/api.js

```diff
@@ -13,7 +13,7 @@ export function initialize(apiUrl) {
 export function authenticate(user) {
   var payload = {
     eauth: 'kubernetes_rbac',
-    username: user.profile.email,
+    username: `oidc:${user.profile.email}`,
```

Comment:
Shouldn't this be done on the Salt eauth side instead?

Reply:
No because Salt eauth module can also authenticate other types of users (especially thinking about the storage-operator ServiceAccount). The UI knows that this user is coming from OIDC, so it seems reasonable to handle it like this.